LibraryApp/src/app/components/Book/styles.js

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+import { FONT_SIZE_LARGE, FONT_SIZE_MEDIUM } from '../../../constants/dimensions';
```

Comment:
You can add relative imports

https://github.com/tleunen/babel-plugin-module-resolver